tests/test_stochastic_fw.py

```diff
@@ -0,0 +1,119 @@
+"""Tests for the Stochastic Frank-Wolfe algorithms."""
```

Comment:
nitpick: I wish these tests ran faster. I tried superficially to speed them up by making alpha only run over 2 numbers instead of 4. No big change though. I guess running on all four variants each time is slow.

Reply:
Sounds good to me!

Reply:
Runs much faster if you reduce the n_samples and n_features. Need to adjust the threshold values accordingly. I can do this if you think it makes sense. Maybe there's some reason I don't see for having higher dimension / more samples in the unit tests.